code/game/dna/dna_modifier.dm

```diff
@@ -134,6 +134,22 @@
 		item.loc = src
 		user.visible_message("\The [user] adds \a [item] to \the [src]!", "You add \a [item] to \the [src]!")
 		return
+
+	else if(istype(item, /obj/item/organ/internal/brain))
+		if (src.occupant)
+			user << "<span class='warning'>The scanner is already occupied!</span>"
+			return
+		var/obj/item/organ/internal/brain/brain = item
+		if(brain.clone_source)
+			user.drop_item()
+			brain.loc = src
+			put_in(brain.brainmob)
```

Comment:
I assume this makes the brainmob the occupant?

Reply:
It does.